p2p/src/sessions/mod.rs

```diff
@@ -166,6 +166,16 @@ where
             .map(|limit| num_outbound_sessions < limit as usize)
             .unwrap_or(true)
     }
+    /// Method to return the diff between limit and outbounds number
+    pub fn get_outbound_diff(&self) -> usize {
+        let num_outbound_sessions = self.get_num_outbound_sessions();
+
+        if let Some(limit) = self.outbound_consolidated.limit {
+            (limit as usize) - num_outbound_sessions
+        } else {
+            1
```

Comment:
Why 1? It could be 1_000_000. But why would someone want unlimited outbound peers?